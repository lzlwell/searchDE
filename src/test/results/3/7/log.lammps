LAMMPS (30 Aug 2012)
units		metal
boundary	p p p
atom_style	atomic
read_data	struct.lammps
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms

pair_style      airebo 3.0
pair_coeff      * * CH.airebo C H

mass	1	12.0
mass	2	1.0

group		fixPosH id <> 1 120
120 atoms in group fixPosH
group		fixPosL id <> 121 239
119 atoms in group fixPosL
group		optPos  id <> 240 1000
61 atoms in group optPos
fix             0       fixPosH setforce NULL 0.0 NULL
fix             1       fixPosH aveforce 0.0 0.0 0.0
fix		2	fixPosL setforce 0.0 0.0 0.0
fix 		3	optPos	setforce NULL 0.0 NULL

neighbor        0.2 bin
neigh_modify	every 1 delay 0 check yes

timestep	0.002
thermo 		1
thermo_style    custom step  pe etotal lx ly lz 

dump		1 all xyz 10000000 dump.atom
dump_modify	1 format "%d  %20.15g  %20.15g  %20.15g" 
#fix 		8 all box/relax iso 0.0 
min_style	cg
minimize	1.0e-12 1.0e-12 10 10
Memory usage per processor = 3.26712 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -1090.0481   -1090.0481      17.1317           10           50 
       1   -1107.1344   -1107.1344      17.1317           10           50 
       2   -1123.3095   -1123.3095      17.1317           10           50 
       3   -1138.0321   -1138.0321      17.1317           10           50 
       4   -1148.2399   -1148.2399      17.1317           10           50 
       5   -1156.1244   -1156.1244      17.1317           10           50 
       6   -1160.4988   -1160.4988      17.1317           10           50 
       7   -1163.9283   -1163.9283      17.1317           10           50 
       8   -1167.5273   -1167.5273      17.1317           10           50 
       9   -1169.6097   -1169.6097      17.1317           10           50 
      10   -1170.7242   -1170.7242      17.1317           10           50 
Loop time of 0.370314 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -1090.04814535     -1169.60965377     -1170.72421173
  Force two-norm initial, final = 209.797 14.8159
  Force max component initial, final = 162.711 9.23655
  Final line search alpha, max atom move = 0.0103832 0.0959047
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.135988 (36.7225)
Neigh time (%) = 0.229752 (62.0425)
Comm  time (%) = 0.00148082 (0.399882)
Outpt time (%) = 0.000144005 (0.0388872)
Other time (%) = 0.00294852 (0.796223)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2449 ave 2449 max 2449 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  45090 ave 45090 max 45090 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 45090
Ave neighs/atom = 150.3
Neighbor list builds = 10
Dangerous builds = 10
min_modify line quadratic

clear
