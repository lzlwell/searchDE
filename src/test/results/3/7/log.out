LAMMPS (30 Aug 2012)
Reading data file ...
  orthogonal box = (0 0 0) to (17.1317 10 50)
  1 by 1 by 1 MPI processor grid
  300 atoms
120 atoms in group fixPosH
119 atoms in group fixPosL
61 atoms in group optPos
Setting up minimization ...
Memory usage per processor = 3.26712 Mbytes
Step PotEng TotEng Lx Ly Lz 
       0   -1090.0481   -1090.0481      17.1317           10           50 
       1   -1107.1344   -1107.1344      17.1317           10           50 
       2   -1123.3095   -1123.3095      17.1317           10           50 
       3   -1138.0321   -1138.0321      17.1317           10           50 
       4   -1148.2399   -1148.2399      17.1317           10           50 
       5   -1156.1244   -1156.1244      17.1317           10           50 
       6   -1160.4988   -1160.4988      17.1317           10           50 
       7   -1163.9283   -1163.9283      17.1317           10           50 
       8   -1167.5273   -1167.5273      17.1317           10           50 
       9   -1169.6097   -1169.6097      17.1317           10           50 
      10   -1170.7242   -1170.7242      17.1317           10           50 
Loop time of 0.370314 on 1 procs for 10 steps with 300 atoms

Minimization stats:
  Stopping criterion = max force evaluations
  Energy initial, next-to-last, final = 
        -1090.04814535     -1169.60965377     -1170.72421173
  Force two-norm initial, final = 209.797 14.8159
  Force max component initial, final = 162.711 9.23655
  Final line search alpha, max atom move = 0.0103832 0.0959047
  Iterations, force evaluations = 10 10

Pair  time (%) = 0.135988 (36.7225)
Neigh time (%) = 0.229752 (62.0425)
Comm  time (%) = 0.00148082 (0.399882)
Outpt time (%) = 0.000144005 (0.0388872)
Other time (%) = 0.00294852 (0.796223)

Nlocal:    300 ave 300 max 300 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Nghost:    2449 ave 2449 max 2449 min
Histogram: 1 0 0 0 0 0 0 0 0 0
Neighs:    0 ave 0 max 0 min
Histogram: 1 0 0 0 0 0 0 0 0 0
FullNghs:  45090 ave 45090 max 45090 min
Histogram: 1 0 0 0 0 0 0 0 0 0

Total # of neighbors = 45090
Ave neighs/atom = 150.3
Neighbor list builds = 10
Dangerous builds = 10
